### PR TITLE
Be more user-friendly with non-existing user v2

### PR DESCRIPTION
**Pull request by lukashino**
Follow-up of https://github.com/OISF/suricata/pull/9628

Redmine ticket:

https://redmine.openinfosecfoundation.org/issues/6278

Describe changes:
- edit the commit message - changed assert to FatalError
- edit error message to hint about options

To use a pull request use a branch name like pr/N where N is the
pull request number.

Alternatively, SV_BRANCH may also be a link to an
OISF/suricata-verify pull-request.

SV_REPO=
SV_BRANCH=pr/1431
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 16234

**Review by jufajardini — COMMENTED** [review]
Error messages looking better, imho! :)

File: src/suricata.c [hunk 1]
```diff
@@ -2157,16 +2157,14 @@ static int InitRunAs(SCInstance *suri)
...
+            FatalError("failed in getting user ID");
         }
 
         sc_set_caps = TRUE;
     /* Get the suricata group ID to given group ID */
     } else if (suri->do_setgid == TRUE) {
         if (SCGetGroupID(suri->group_name, &suri->groupid) != 0) {
-            SCLogError("failed in getting group ID");
-            return TM_ECODE_FAILED;
+            FatalError("failed in getting group ID");
```
Comment (jasonish) [inline src/suricata.c:2167]: While still valid enlighs, "failed to get user ID" does read better IMO.

**Comment by lukashino** [comment]
continues in https://github.com/OISF/suricata/pull/9690

File: src/suricata.c (hunk 1 above)
Comment (lukashino) [reply src/suricata.c:2167]: changed

**Pull request closed without merging**